src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/zip/ZipStep.java

```diff
@@ -34,6 +34,7 @@
 import org.jenkinsci.plugins.workflow.steps.StepDescriptor;
 import org.jenkinsci.plugins.workflow.steps.StepExecution;
 import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
```

Comment:
I do not see a reason to add an import that is not used. This seems like leftover code from an earlier revision. Suggest removing the addition of this import.

Reply:
Done.